engine/src/main/java/org/terasology/engine/TerasologyEngine.java

```diff
@@ -114,7 +114,7 @@
 
     private static final Logger logger = LoggerFactory.getLogger(TerasologyEngine.class);
 
-    private static final int ONE_MEBIBYTE = 1024 * 1024;
```

Comment:
This point here is that 1024 * 1024 is easier to understand. The actual value is computed at "compile time" so there's no overhead involved.